app/serializers/authentication_serializer.rb

```diff
@@ -0,0 +1,3 @@
+class AuthenticationSerializer: ApplicationSerializer
+
+end
```

Comment:
We can define it within 1 line 

```suggestion
class AuthenticationSerializer: ApplicationSerializer; end
```